api/src/main/kotlin/org/jellyfin/apiclient/api/client/extensions/UserApiExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.apiclient.api.client.extensions
+
+import org.jellyfin.apiclient.api.operations.UserApi
+import org.jellyfin.apiclient.model.AuthenticateUserByName
+
+/**
+ * Extension function for the authenticateUserByName operation that accepts the username and password directly
+ */
+suspend inline fun UserApi.authenticateUserByName(username: String, password: String) = authenticateUserByName(
+	data = AuthenticateUserByName(
+		username = username,
+		password = password
+	)
```

Comment:
Not sure whether you have to specify every parameter name directly, but ok ^^

Reply:
The reason I did that is because `AuthenticateUserByName` is generated by the generator and if it (for some reason) changes the compiler will complain to us.
It also skips the `pw` property (because yes we have password and pw) which if I remember correctly is the second parameter, defaulting to null.

Reply:
Ah, that makes sense then. I remember that pw thing from talks inside the chat a few days ago.